bindings/rust/wasmedge-sys/src/instance/memory.rs

```diff
@@ -37,23 +70,83 @@ impl Memory {
         }
     }
 
-    /// Copy the data to the output buffer from a memory instance.
-    pub fn get_data(&self, offset: u32, size: u32) -> WasmEdgeResult<impl Iterator<Item = u8>> {
-        let mut data = Vec::with_capacity(size as usize);
+    /// Copies the data from the [`Memory`] to the output buffer.
+    ///
+    /// # Arguments
+    ///
+    /// - `offset` specifies the data start offset in the [`Memory`].
+    ///
+    /// - `len` specifies the requested data length.
+    ///
+    /// # Errors
+    ///
+    /// If the `offset` + `len` is larger than the data size in the [`Memory`], then an error is returned.
+    ///
+    /// # Examples
+    ///
```

Comment:
There is no example here?

Reply:
Accepted. Removed the Example tag. Thanks!